api/src/main/java/edu/cornell/mannlib/vitro/webapp/auth/checks/CheckFactory.java

```diff
@@ -48,9 +49,18 @@ public static Check createCheck(QuerySolution qs, AttributeValueKey dataSetKey)
                 check = null;
         }
         check.setType(CheckType.valueOf(testId));
+        setConfiguration(qs, check);
         return check;
     }
 
+    private static void setConfiguration(QuerySolution qs, Check check) {
+        RDFNode rdfNode = qs.get("config");
+        if (rdfNode == null || !rdfNode.isLiteral()) {
```

Comment:
As I mentioned, you can switch to using Objects.nonNull() if you have time and will to do it :smile:

Reply:
It is better to leave null checks consistent in the code base, not have it Objects.isNull(var) in one place and var == null in another. I would leave it as it is.

Reply:
We can switch to the better practice now, and maybe in the future refactor the entire code base? What do you think?

Reply:
I wouldn't call it a better practice. https://community.sonarsource.com/t/set-usage-of-objects-isnull-and-objects-nonnull-as-code-smells/58469

Reply:
Ok, you are right :smile: I didn't know that it is, in fact, a code smell. Sorry.